src/igor/sub.go

```diff
@@ -69,7 +71,7 @@ func init() {
 	cmdSub.Flag.StringVar(&subI, "i", "", "")
 	cmdSub.Flag.IntVar(&subN, "n", 0, "")
 	cmdSub.Flag.StringVar(&subC, "c", "", "")
-	cmdSub.Flag.IntVar(&subT, "t", 60, "")
+	cmdSub.Flag.StringVar(&subT, "t", "60m", "")
```

Comment:
Too bad we want to support days otherwise we could just use `flag.Duration`. Here's why `time.ParseDuration` does not support days: https://github.com/golang/go/issues/11473.

Reply:
Yeah I read that. At the end of the day it smacks to me like an arbitrary cut-off that they simply don't want to budge from, given the leap-second argument still applies at the hour increment. I'm not too worried as we repeat back the reservation end time which the user can confirm to be correct, however you are right that it should be expressly noted in the help text.